lib/manager/ansible-galaxy/collections.ts

```diff
@@ -0,0 +1,161 @@
+import * as datasourceGalaxyCollection from '../../datasource/galaxy-collection';
+import * as datasourceGitTags from '../../datasource/git-tags';
+import * as datasourceGithubTags from '../../datasource/github-tags';
+import { SkipReason } from '../../types';
+import { PackageDependency } from '../types';
+import { blockLineRegEx, galaxyDepRegex, newBlockRegEx } from './util';
+
+function interpretLine(
+  lineMatch: RegExpMatchArray,
+  lineNumber: number,
+  dependency: PackageDependency
+): PackageDependency {
```

Comment:
```suggestion
): void {
```

use `void` if returned value is never used